packages/calcite-components/src/components/chip/chip.scss

```diff
@@ -14,6 +14,19 @@
   .container.is-circle {
     @apply w-6 h-6;
   }
+
+  .multiple {
+    .select-icon {
+      padding-inline: var(--calcite-spacing-xxs);
```

Comment:
Could you look at simplifying by using internal CSS props to set the appropriate spacing values?  Something similar to [this](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/components/segmented-control-item/segmented-control-item.scss#L98-L124)?